src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -2285,5 +2285,170 @@ static class ArrayExtensions
+
+            var syntaxTree = comp.SyntaxTrees[0];
+            var root = syntaxTree.GetRoot();
+            var model = comp.GetSemanticModel(syntaxTree);
+
+            var memberAccess = root.DescendantNodes().OfType<MemberAccessExpressionSyntax>().ToList();
+
+            var symInfo = model.GetSymbolInfo(memberAccess[0]);
+            Assert.Equal(PublicNullableAnnotation.Annotated, ((IFieldSymbol)symInfo.Symbol).NullableAnnotation);
+            Assert.Equal(PublicNullableAnnotation.Annotated, (symInfo.Symbol.ContainingType).TypeArgumentNullableAnnotations[0]);
+            symInfo = model.GetSymbolInfo(memberAccess[1]);
+            Assert.Equal(PublicNullableAnnotation.NotAnnotated, ((IFieldSymbol)symInfo.Symbol).NullableAnnotation);
+            Assert.Equal(PublicNullableAnnotation.NotAnnotated, (symInfo.Symbol.ContainingType).TypeArgumentNullableAnnotations[0]);
+        }
+
+        [Fact]
+        public void GetSymbolInfo_EventAssignmentSymbols()
+        {
+            var source = @"
+#pragma warning disable CS0067
+using System;
+class C<T>
+{
+    public event EventHandler? Event;
+
+    static C<U> Create<U>(U u) => new C<U>();
+
+    static void M(object? o)
+    {
+        var c1 = Create(o);
+        c1.Event += (obj, sender) => {};
+        if (o is null) return;
+        var c2 = Create(o);
+        c2.Event += (obj, sender) => {};
+        c2.Event += c1.Event;
+    }
+}";
+            var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
+            comp.VerifyDiagnostics();
+
+            var syntaxTree = comp.SyntaxTrees[0];
+            var root = syntaxTree.GetRoot();
+            var model = comp.GetSemanticModel(syntaxTree);
+
+            var memberAccess = root.DescendantNodes().OfType<MemberAccessExpressionSyntax>().ToList();
+
+            var symInfo = model.GetSymbolInfo(memberAccess[0]);
+            Assert.Equal(PublicNullableAnnotation.Annotated, ((IEventSymbol)symInfo.Symbol).NullableAnnotation);
+            Assert.Equal(PublicNullableAnnotation.Annotated, (symInfo.Symbol.ContainingType).TypeArgumentNullableAnnotations[0]);
+            symInfo = model.GetSymbolInfo(memberAccess[1]);
+            Assert.Equal(PublicNullableAnnotation.Annotated, ((IEventSymbol)symInfo.Symbol).NullableAnnotation);
+            Assert.Equal(PublicNullableAnnotation.NotAnnotated, (symInfo.Symbol.ContainingType).TypeArgumentNullableAnnotations[0]);
+
+            var event1 = model.GetSymbolInfo(memberAccess[2]).Symbol;
+            var event2 = model.GetSymbolInfo(memberAccess[3]).Symbol;
+            Assert.NotNull(event1);
+            Assert.NotNull(event2);
+            Assert.True(event1.Equals(event2, SymbolEqualityComparer.Default));
+            Assert.False(event1.Equals(event2, SymbolEqualityComparer.IncludeNullability));
+        }
+
+        [Fact]
+        public void GetSymbolInfo_EventAccessSymbols()
```

Comment:
>EventAccessSymbols [](start = 34, length = 18)

Perhaps this should be "EventAssignmentSymbols" and the test above "EventAddSymbols".

Reply:
I went by the underlying BoundNodes, which was `BoundEventAccess` directly for this test and `BoundEventAssignment` via the synthesized `BoundEventAccess` I adjusted for in the semantic model for the above test. But I can rename.

---
In reply to: [319283902](https://github.com/dotnet/roslyn/pull/38354#discussion_r319283902) [](ancestors = 319283902)